editor/src/components/canvas/ui-jsx-canvas.spec.tsx

```diff
@@ -1308,6 +1308,7 @@ export var storyboard = (
             <div
               class=\\"ant-picker\\"
               style=\\"width: 123px; height: 51px; left: 113px; top: 395px;\\"
+              data-uid=\\"antd-date-picker\\"
```

Comment:
most interesting – do you know how did this start working on this branch?

Reply:
I didn't look into this very far at all mostly because it seemed like a step up from what we had.

Reply:
you very likely fixed https://github.com/concrete-utopia/utopia/issues/1020

Reply:
I assumed it was never in there in the first place, so that's actually even better news.